static/js/subs.js

```diff
@@ -785,6 +785,17 @@ $(function () {
         var search_term = user_list.expectOne().val().trim();
         var search_terms = search_term.toLowerCase().split(",");
 
+        // create a map of labels
+        var users_map = $('#user-checkboxes')
```

Comment:
@neerajwahi Rerendering `new_stream_users` with new set of filtered user will also require to keep track of users that are already checked. While rerendering if user has already checked a large number of users, it will cause the same problem that we will have to iterate over whole filtered userlist to check users using `$("label[for='" + user.email + "']")` which is very slow. Thoughts?